src/server/templates/page.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function page({ body = "", assets, title }) {
+  return `
```

Comment:
I know this file is just to get the ball rolling(!) but I'd be curious to know if this is the plan for how we'll be writing all HTML templates or if we should investigate a Hogan / Handlebars alternative.